Porblem2.py

```diff
@@ -0,0 +1,29 @@
+"""
+15. 3Sum
+Time Complexity - O(n^2)
+Space Complexity - O(1)
+Fixed a pointer and then perform 2 pointer approach on entire list and append values in output
+"""
```

Comment:
Good Job Diksha! with all the problems. Please try to add more comments.Adding comments while writing code may help streamline your thought process.